config/samples/vela-server/Demo.md

```diff
@@ -0,0 +1,145 @@
+# Vela Server
```

Comment:
This shouldn't be called `vela server`. Probably `application`?

Reply:
Agree, I will edit it